faulty.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   # Other non-essential development dependencies go in the Gemfile.
   spec.add_development_dependency 'connection_pool', '~> 2.0'
   spec.add_development_dependency 'json'
-  spec.add_development_dependency 'redis', '>= 3.0'
+  spec.add_development_dependency 'redis', '>= 4.0'
```

Comment:
Do we need to drop support for 3? That's going to be a breaking change.

Reply:
3 wasn't running in CI, so I opted to drop it instead of adding it there. But you're right, that would warrant a separate PR.